### DISCUSSION TITLE
N/A

### DISCUSSION THREAD

**Discussion by dcummuta** [nextgenhealthcare/connect#5028] [Q&A]

**Comment by pacmano1** [comment]
That is because the rest of it is fundamental mirth stuff, which is a broad topic for this post.

I'd suggest taking a look at https://github.com/nextgenhealthcare/connect/discussions/4659 for potential training resources.

**Reply by dcummuta** [reply]
I removed my question since it is too fundamental.  Sorry for bothering you.

**Reply by pacmano1** [reply]
It is not a bother at all, I  am  just suggesting that there is foundational mirth stuff that is important to understand prior to trying code from the Internet.    There are probably half a dozen different ways to grab the data collected for downstream use, but it all depends on your use case.
